### DISCUSSION TITLE
Does ISCE show the original resolution of processing data?

### DISCUSSION THREAD

**Discussion by remotesensingvn** [isce-framework/isce2#627] [Q&A]
Hello,

I'm processing ERS/ENV, ALOS, Sentinel-1 and wondering does ISCE show the spatial resolution (azimuth x range) of image?

The information on the internet is varying the resolution and causing misunderstood of the real resolution in order to generate proper results with downlooking. Is there anyone knowing this information?

Thank you.

**Comment by fdelgadodelapuente** [comment]
for stripmapApp.py read the section Example 1: (extract metadata, range and azimuth pixel size)
https://github.com/isce-framework/isce2-docs/blob/master/Notebooks/UNAVCO_2020/Stripmap/stripmapApp.ipynb



For Sentinel-1 it is 14.1 in azimuth and 2.3 in slant range
